plio/tests.py

```diff
@@ -139,18 +310,244 @@ class ItemTestCase(BaseTestCase):
     def setUp(self):
         super().setUp()
 
-    def test_for_item(self):
-        # write API calls here
-        self.assertTrue(True)
+        # seed a video
+        self.video = Video.objects.create(
+            title="Video 1", url="https://www.youtube.com/watch?v=vnISjBbrMUM"
+        )
+        # seed a plio
+        self.plio = Plio.objects.create(
+            name="Plio", video=self.video, created_by=self.user
+        )
+        # seed an item
+        self.item = Item.objects.create(type="question", plio=self.plio, time=1)
+
+    def test_guest_cannot_list_items(self):
+        # unset the credentials
+        self.client.credentials()
+        # get items
+        response = self.client.get(reverse("items-list"))
+        self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)
+
+    def test_user_can_list_items(self):
+        # get items
+        response = self.client.get(reverse("items-list"))
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(len(response.data), 1)
+
+    def test_user_list_items_by_plio(self):
+        # create a new plio
+        new_plio = Plio.objects.create(
+            name="Plio 2", video=self.video, created_by=self.user
+        )
+        # attach a new item to the new plio
+        Item.objects.create(type="question", plio=new_plio, time=1)
+
+        # get items
+        response = self.client.get(reverse("items-list"), {"plio": self.plio.uuid})
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+        # the number of responses should still be 1 - the item created above
+        # should not be included
+        self.assertEqual(len(response.data), 1)
+
+    def test_user_list_own_items(self):
+        """A user should only be able to list their own items"""
+        # create plio from a different user
+        new_plio = Plio.objects.create(
+            name="Plio 2", video=self.video, created_by=self.user_2
+        )
+
+        # create item from a different user
+        Item.objects.create(type="question", plio=new_plio, time=1)
+
+        # get items
+        response = self.client.get(reverse("items-list"))
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        # the count should remain 1 as the new item was created with different user
+        self.assertEqual(len(response.data), 1)
+
+    def test_user_can_list_own_items_org(self):
```

Comment:
```suggestion
    def test_org_user_can_list_own_items(self):
```
I think this is more clear